### PR TITLE
[Synfig Studio] Fixed incorrect import of `lst` files when path contains non-Latin characters

### PR DESCRIPTION
**Pull request by ice0**
Fix https://github.com/synfig/synfig/issues/1898

### CODE REVIEW

File: synfig-core/src/synfig/listimporter.cpp [hunk 1]
```diff
@@ -71,10 +67,11 @@ Importer(identifier)
 {
 	fps=15;
 
-	ifstream stream(identifier.filename.c_str());
+	FileSystem::ReadStream::Handle stream = identifier.get_read_stream();
 
-	if(!stream)
-	{
+	// TODO(ice0): There is no way to report about error to GUI. We can't
+	//  throw error from constructor (bad practice), and can't return error code.
```
Comment (rodolforg) [inline synfig-core/src/synfig/listimporter.cpp:73]: Agree. Should we refactor the Importer class to have a `load()` method – that should be called only once?

**Review by rodolforg — COMMENTED** [review]
I can't test it because I barely have access to Windows on those pandemic days (as I'm home office'ing due to social distancing).
Besides, my language, Portuguese, uses latin-only characters XD

File: synfig-core/src/synfig/listimporter.cpp (hunk 1 above)
Comment (ice0) [reply synfig-core/src/synfig/listimporter.cpp:73]: I think yes, this is a good idea. We can back to this problem after 1.4.1 release.

**Comment by ice0** [comment]
> I can't test it because I barely have access to Windows on those pandemic days (as I'm home office'ing due to social distancing).
> Besides, my language, Portuguese, uses latin-only characters XD

No problem here, I already tested it and all works fine. Just need someone who can check the code too.

**Review by rodolforg — APPROVED** [review]
Code looks ok to me

**Pull request merged**